test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -735,8 +828,12 @@ class test_uct_cm_sockaddr : public uct_test {
     bool                   m_server_start_disconnect;
     bool                   m_delay_conn_reply;
     std::queue<uct_conn_request_h> m_delayed_conn_reqs;
+    size_t                 short_priv_data_len, long_priv_data_len;
+    std::vector<char>      short_priv_data;
+    std::vector<char>      long_priv_data;
```

Comment:
did not we agree to move it to separate static class instance?

Reply:
after discussing with @yosefe decided to make the client and server pack_cb functions non static and move them to the test_uct*_sockaddr classes.
seems that it makes things simpler, wdyt?

Reply:
C++ non-static methods can't be passed as C-callbacks since they have implicit `this` argument

Reply:
i meant static in the entity class which is used not only for client-server testing.
client_cm_priv_data_cb and server_cm_priv_data_cb are static in the test_uct_cm_sockaddr class, where they are used.

Reply:
ok
this conversation related to https://github.com/openucx/ucx/pull/5073/commits/199466bf155a43ef879ac47fc06f8e5e9864c0c8#r424965572, resolving this one to have a single place.